packages/esm-service-queues-app/src/queue-table/queue-entry-actions/queue-entry-confirm-action.modal.tsx

```diff
@@ -85,7 +85,7 @@ export const QueueEntryConfirmActionModal: React.FC<QueueEntryUndoActionsModalPr
         ) : (
           <Stack gap={4}>
             <h5>{queueEntry.display}</h5>
-            <p>{modalInstruction}</p>
+            <span>{modalInstruction}</span>
```

Comment:
This fixes a validateDOMNesting warning: `<p> cannot appear as a descendant of <p>`.